app/services/sufia/workflow/workflow_factory.rb

```diff
@@ -0,0 +1,7 @@
+module Sufia
+  module Workflow
+    class WorkflowFactory < CurationConcerns::Workflow::WorkflowFactory
+      self.workflow_strategy = WorkflowByAdminSetStrategy
```

Comment:
I was thinking we could just set `CurationConcerns::Workflow::WorkflowFactory.workflow_strategy = Sufia::WorkflowByAdminSetStrategy`